extensions/kubectl-cli/src/cli-run.ts

```
@@ -68,34 +57,45 @@ export async function installBinaryToSystem(binaryPath: string, binaryName: stri
-    args = [`"${binaryPath}"`, `"${destinationPath}"`];
+    // admin privileges are are not needed on Windows
+    await fs.promises.copyFile(binaryPath, destinationPath);
+    return destinationPath;
   } else if (system === 'darwin') {
     command = 'exec';
-    args = ['cp', binaryPath, destinationPath];
+    args = ['cp', '-f', binaryPath, destinationPath];
   } else if (system === 'linux') {
     command = '/bin/sh';
     args = ['-c', `cp ${binaryPath} ${destinationPath}`];
   }
```

Biome (2.5.3)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)